sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/AnalyzeLayout.java

```diff
@@ -52,20 +56,26 @@ public static void main(final String[] args) throws IOException {
                 documentPage.getUnit());
 
             // lines
-            documentPage.getLines().forEach(documentLine ->
-                System.out.printf("Line %s is within a bounding box %s.%n",
+            documentPage.getLines().forEach(documentLine -> {
+                System.out.printf("Line '%s' is within a bounding box %s.%n",
                     documentLine.getContent(),
-                    documentLine.getBoundingBox().toString()));
+                    documentLine.getBoundingBox().toString());
+
+                List<DocumentWord> containedWords = getWordsInALine(documentLine, documentPage.getWords());
```

Comment:
I thought we were going to add standalone samples to showcase this? If we add it here, I think there could be confusion for our data gathering purposes about whether the customer was simply looking at a sample about how to analyze layout vs looking for how to use the spans to find related elements.

Reply:
yeah, I was a bit confused. But yes, for telemetry reasons I think it will make sense to have this as a standalone sample.